README.md

```diff
@@ -1 +1,41 @@
 # data-integrity-test-suite-assertion
+
+## Table of Contents
+
+- [Background](#background)
+- [Install](#install)
+- [Test](#test)
+- [Usage](#usage)
+- [Implementation](#implementation)
+
+## Background
+This library exports a function `checkDataIntegrity()` that can be called in
+other test suites (such as Ed25519, BBS+, etc.) to validate the structure of
+the proofs on a document.
+
+## Install
+
+```js
+npm i
+```
+
+## Test
+
+```js
+npm test
+```
+
+## Usage
+
+```js
+// Require `data-integrity-test-suite-assertion` in the test suite
+const {checkDataIntegrity} = require('data-integrity-test-suite-assertion');
+
+const implementation = new Implementation(issuer);
```

Comment:
Examples should be runnable. `Implementation` and `issuer` is not defined. Additionally, this test suite should not be tightly bound to "issuer" and "verifier" language. Any document can be signed and verified using the data integrity specification, not just verifiable credentials.

Reply:
Updated the example to not include implementation/issuer language [c039eab](https://github.com/digitalbazaar/data-integrity-test-suite-assertion/pull/1/commits/c039eab3e636847943d10f0dda314fc86971bea3)